src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -733,6 +792,41 @@ public void setNrAudioChannels(int numberOfChannels) {
 	}
 
 	/**
+	 * Sets the number of audio channels.
+	 *
+	 * @param numberOfChannels the number of channels to set, use 0 if unknown.
+	 * @since 6.7.2
+	 */
+	@SuppressWarnings("deprecation")
+	public void setNumberOfChannels(int numberOfChannels) {
+		this.numberOfChannelsArray = new int[] {numberOfChannels};
+		this.nrAudioChannels = numberOfChannels;
+		audioProperties.setNumberOfChannels(numberOfChannels);
+	}
+
+	/**
+	 * Sets an array of audio channels.
+	 *
+	 * @param numberOfChannelsArray the array with the number of channels to
+	 *            set, use {@code null} or an empty array if unknown.
+	 * @since 6.7.2
+	 */
+	@SuppressWarnings("deprecation")
+	public void setNumberOfChannels(int[] numberOfChannelsArray) {
+		if (numberOfChannelsArray == null) {
+			this.numberOfChannelsArray = new int[0];
+		} else {
+			if (this.numberOfChannelsArray == null || this.numberOfChannelsArray.length != numberOfChannelsArray.length) {
+				this.numberOfChannelsArray = new int[numberOfChannelsArray.length];
+			}
+			System.arraycopy(numberOfChannelsArray, 0, this.numberOfChannelsArray, 0, numberOfChannelsArray.length);
+		}
+		// Find the highest number to store in the deprecated fields
```

Comment:
I'm storing the value in the deprecated fields as well as in the new array. That way, if a plugin reads one of the deprecated fields directly, it will still get a result (even if it's less accurate). "deprecated" is because the referred fields are annotated with ```@Deprecated```, and are no longer the "current" place to store the information.